src/transformers/quantizers/quantizer_bnb_4bit.py

```diff
@@ -58,8 +58,6 @@ def __init__(self, quantization_config, **kwargs):
             self.modules_to_not_convert = self.quantization_config.llm_int8_skip_modules
 
     def validate_environment(self, *args, **kwargs):
-        if not torch.cuda.is_available():
-            raise RuntimeError("No GPU found. A GPU is needed for quantization.")
         if not is_accelerate_available():
             raise ImportError("Using `bitsandbytes` 4-bit quantization requires Accelerate: `pip install accelerate`")
```

Comment:
Somewhat unrelated to this PR, but: why is this the case? 🤔

Reply:
Not sure about it.